### DISCUSSION TITLE
Interactive Lorenz Attractor

### DISCUSSION THREAD

**Discussion by dvd101x** [josdejong/mathjs#3147] [Ideas]
Hi, I was thinking of making the [Lorenz attractor](https://github.com/josdejong/mathjs/blob/develop/examples/browser/lorenz.html) interactive.

Would it be OK if I make a PR similar to this?

![imagen](https://github.com/josdejong/mathjs/assets/16261897/6e8f5416-0aeb-4c03-aba6-9b5abce45259)

**Comment by josdejong** [comment]
Nice idea! Maybe put this as a second example, so we keep the simple (current example) as is?

**Reply by dvd101x** [reply]
Good idea. I will make a separate example, if the code ends up being simple enough, we could reevaluate if a it should be done on the same file.
